lib/db/mongo.js

```diff
@@ -11,13 +11,15 @@ function mongoDB(options) {
 
   console.log('Starting MongoDB...');
 
-  // this.open();
+  this.dburi = options.dburi;
+  this.dbname = options.dbname;
+  this.connectionString = `mongodb://${this.dburi}/${this.dbname}`;
```

Comment:
I'd call dburi dbHost, since the whole connection string is the uri, and maybe add on any additional options as an extra params/options component at the end.